### PR TITLE
requireLineFeedAtFileEnd: make use of assert

### PR DESCRIPTION
**Pull request by gero3**

### CODE REVIEW

**Comment by mikesherov** [comment]
Looks goods, just needs more tests: 2 tests in JsFile, 1 for each function you added. 

Thanks again @gero3!

File: lib/rules/require-line-feed-at-file-end.js [hunk 1]
```diff
@@ -34,10 +34,14 @@ module.exports.prototype = {
...
 
     check: function(file, errors) {
-        var lines = file.getLines();
-        if (lines[lines.length - 1] !== '') {
-            errors.add('Missing line feed at file end', lines.length, 0);
-        }
+        var lastToken = file.getLastFileToken();
+        var prevToken = file.getPrevToken(lastToken, {includeComments: true});
+        errors.assert.linesBetween({
+                token: prevToken,
```
Comment (mikesherov) [inline lib/rules/require-line-feed-at-file-end.js]: This and the next 4 lines are over indented.

File: lib/rules/require-line-feed-at-file-end.js [hunk 2]
```diff
@@ -34,10 +34,14 @@ module.exports.prototype = {
...
-        if (lines[lines.length - 1] !== '') {
-            errors.add('Missing line feed at file end', lines.length, 0);
-        }
+        var lastToken = file.getLastFileToken();
+        var prevToken = file.getPrevToken(lastToken, {includeComments: true});
+        errors.assert.linesBetween({
+            token: prevToken,
+            nextToken: lastToken,
+            atLeast: 1,
+            message: 'Expected a padding newline before closing curly brace'
```
Comment (hzoo) [inline lib/rules/require-line-feed-at-file-end.js]: Error message should be "Missing line feed at file end" still haha?

Comment (gero3) [inline lib/rules/require-line-feed-at-file-end.js]: fixed

File: lib/js-file.js [hunk 3]
```diff
@@ -191,6 +191,24 @@ JsFile.prototype = {
     },
 
     /**
+     * Returns the first token for the file from the AST.
+     *
+     * @returns {Object}
+     */
+    getFirstFileToken: function() {
```
Comment (mikesherov) [inline lib/js-file.js]: One last nit. This should be called `getFirstToken` instead, because the `File` is redundant. Same goes for `getLastToken`.

Comment (gero3) [inline lib/js-file.js]: fixed

**Comment by gero3** [comment]
Is there something wrong with this pull because appveyor failed??

**Pull request closed without merging**